src/core/plugins/syntax-highlighting/wrap-components/SyntaxHighlighter.jsx

```diff
@@ -8,7 +8,7 @@ import get from "lodash/get"
 const SyntaxHighlighterWrapper = (Original, system) => {
   const SyntaxHighlighter = ({ renderPlainText, children, ...rest }) => {
     const configs = system.getConfigs()
-    const canSyntaxHighlight = !!get(configs, "syntaxHighlight.activated")
+    const canSyntaxHighlight = get(configs, "syntaxHighlight.activated")
```

Comment:
Should we change this to
```js
const { syntaxHighlight } = system.getConfigs()
const canSyntaxHighlight = syntaxHighlight && syntaxHighlight.activated
```

And then do the same thing for `syntaxHighlight.theme`:
https://github.com/swagger-api/swagger-ui/blob/f7b9f8ddf1fe4ce2d285032b1a3834efd9be4321/src/core/plugins/syntax-highlighting/components/SyntaxHighlighter.jsx#L16-L17

Reply:
Yes we can, I would suggest following:

```js
const canSyntaxHighlight = !!system.getConfigs().syntaxHighlight?.activated
```

We have to typecast again here because of the fact that `syntaxHighlight` can be defined as `false`.

Reply:
> And then do the same thing for syntaxHighlight.theme:

yes, sure